src/main.rs

```diff
@@ -169,60 +170,63 @@ impl Application {
 
     async fn on_text(&self, message: Message) -> Result<(), AudioPlayerError> {
         let msg = message.text;
-        if msg.starts_with("!") {
-            let tokens = msg[1..].split_whitespace().collect::<Vec<_>>();
+        if msg.starts_with("poke") {
```

Comment:
pokeadd <url>? what is the syntax now?

Reply:
It's `poke add <url>` to add to add now.
Anything that starts with poke works though so you could type pokebot too if you wanted to.